Sofascore/Sofascore/TeamScoreView.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class TeamScoreView: BaseView {
+    
+    let teamScore: UILabel = .init()
+    
+    override func addViews() {
+        addSubview(teamScore)
+    }
+    
+    override func styleViews() {
+        teamScore.font = Fonts.body
+    }
+    
+    override func setupConstraints() {
+        teamScore.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+            $0.width.equalTo(32)
```

Comment:
```suggestion
        snp.makeConstraints {
            $0.width.equalTo(32)
        }
        teamScore.snp.makeConstraints {
            $0.edges.equalToSuperview()
```

Probaj ovako, mislim da bi bilo na view koji je superview za teamScore labelu postaviti width pa da se ona prema njemu ravna.